software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -76,10 +77,13 @@ public void customize() {
 
     @Override
     public void launch() {
-        executeCommandInTask(
```

Comment:
Let's look at that in a separate PR. Making both optional (neither obligatory) would be fine with me.